src/app/course/courses-shared/course-authors/course-authors.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroupDirective } from '@angular/forms';
+
+@Component({
+  selector: 'app-course-authors',
+  templateUrl: './course-authors.component.html',
+  styleUrls: ['./course-authors.component.less']
+})
```

Comment:
all dumb component should have onPush strategy. Including course-item as it is a requirement in the hometask